pkg/api/helpers_test.go

```diff
@@ -342,102 +341,6 @@ func TestMatchTaint(t *testing.T) {
 	}
 }
 
-func TestGetAvoidPodsFromNode(t *testing.T) {
```

Comment:
the same test already exists in pkg/api/v1/helpers_test.go on the versioned helper